charts/kube-aws/rbac-admin.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
I am unable to find tha k8-service-kill.png in icons

Reply:
okay added that!